### DISCUSSION TITLE
Change name of is_updated to better reflect its use case

### DISCUSSION THREAD

**Discussion by TiCampbell** [Get-Your/Get-Your#222] [Ideas]
I propose changing the name of `is_updated` (in all tables, to match) to better reflect its use case of letting the program coordinators know information needs to be updated in their own systems. Since this is a business use, only updates to data they have access to need to be reflected in `is_updated`; currently in `Address`, only a change in `mailing_address_id` triggers `is_updated`. Any changes in non-business-use data are governed by internal processes - in the same example, `eligibility_address_id` is only set/changed for new accounts and renewals, both of which send all data to the income-verification team and therefore ignore `is_updated` anyway.

Maybe something like `has_business_updates` or something relating to 'external' would be clearer.

@penoptech what do you think?

**Comment by penoptech** [comment]
I agree. `has_business_updates` better reflects the specific use case of informing program coordinators about the need for data updates in their own systems. `is_updated` was somewhat ambiguous to begin with and the new column will help add clarity to that data

**Comment by TiCampbell** [comment]
This was (long ago) added to the enhancements list (#254).
